app/src/main/java/com/realityexpander/tasky/agenda_feature/data/common/typeParceler/TextStyleParceler.kt

```diff
@@ -0,0 +1,45 @@
+package com.realityexpander.tasky.agenda_feature.data.common.typeParceler
+
+import android.os.Parcel
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import kotlinx.parcelize.Parceler
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import logcat.logcat
+
+// Note: Only saves the `fontSize` and `fontWeight` of the TextStyle
+object TextStyleParceler : Parceler<TextStyle> {
```

Comment:
Why do you need to save fontSize and fontWeight of a Text?

Reply:
Im using a general "edit text" composable, for both `title` and `description` (or any other text) and I want the editor to match the content `size` and `weight`. So, this must be saved in the`savedStateHandle` `bundle` as a `parcel`, and the `TextStyle` does not natively support `@Parcelize`, so I had to write a custom one that saves the two important settings for the edit text dialog.

Reply:
OK, but I want the editor to be able to set any text style, and the call site defines this.